src/docker/Volumes.ts

```diff
@@ -14,15 +14,17 @@ export interface DockerVolume extends DockerObject {
     readonly Description?: string;
 }
 
+export interface VolumeInspectionContainers {

```

Comment:
Slightly more interesting change...defined and exported this here so it can be referenced in `DockerodeUtils.ts`